core/new-gui/src/app/dashboard/component/feature-container/saved-workflow-section/ngbd-modal-workflow-executions/ngbd-modal-workflow-executions.component.html

```diff
@@ -28,6 +28,7 @@ <h4 class="modal-title">Execution results of this workflow</h4>
           ></i>
         </td>
         <td nzEllipsis>{{(i + 1)}}</td>
+        <td nzEllipsis>{{row.uId || "N/A" }}</td>
```

Comment:
maybe "anonymous" instead of N/A. Also please check the functionality when the "User System" is disabled and set to `false`. That case, should also work with the current implementation of `dummy_wid_uid`